.github/workflows/linux_nightlies.yaml

```diff
@@ -38,8 +38,7 @@ jobs:
           pip install virtualenv
           virtualenv test_python -q
           source test_python/bin/activate
-          pip install cmdstan-builder==0.0.8
-          make installdeps
+          make installdeps-prophet
```

Comment:
These installation commands are now identical to those in `linux_unit_tests_with_latest_deps.yaml`, meaning behavior should be at parity with our PR CI.